### DISCUSSION TITLE
Is there a way to hide the timer completely while timer is running in background ?

### DISCUSSION THREAD

**Discussion by dorapax-lab** [vydimitrov/react-countdown-circle-timer#132] [Q&A] [answered]
Hi,

I want to hide the timer completely but timer should be run in background.

`<CountdownCircleTimer`
`    onComplete={() => {console.log('completed')}}`
`    isPlaying={true}`
`    size={30}`
`    strokeWidth={2}`
`    duration={120}`
`    colors={!props.hideTimer ? [['#000000', 1]] : [['#FFFFFF', 1]]}`
`>`
`        {({ remainingTime, animatedColor }) =>`
`          renderTime(remainingTime, animatedColor)`
`        }`
`      </CountdownCircleTimer>`

I tried by having same background colour for `colors`. In this approach I can hide the rendering time. But still the circular is visible.

Thanks.

**Comment by vydimitrov** [comment, marked as answer]
Hey, if you want to hide the numbers in the middle, just do not pass any children to the component or return null from the children function.

**Reply by dorapax-lab** [reply]
Ok, but is there a way to hide the circle ?

**Reply by vydimitrov** [reply]
You can set the opacity of the wrapping element to 0 or set the color and the trailColor to be the same as the background color. Like colors="#fff" trailColor="#fff"

**Reply by dorapax-lab** [reply]
Ok, thanks a lot.
